core/modules/file.py

```diff
@@ -81,7 +81,7 @@ class InjectStoreURLBone(BaseBone):
     def unserialize(self, skel, name):
         if "dlkey" in skel.dbEntity and "name" in skel.dbEntity:
             skel.accessedValues[name] = utils.downloadUrlFor(
-                skel["dlkey"], skel["name"], derived=False, expires=conf["viur.downloadUrlFor.expiration"]
+                skel["dlkey"], skel["name"], derived=False, expires=conf["viur.render.json.downloadUrlExpiration"]
```

Comment:
Hmm. The correct replacement of _viur.downloadUrlFor.expiration_ would be _viur.render.html.downloadUrlExpiration_... 
But actually this a third case. _viur.downloadUrlFor.expiration_ or _viur.InjectStoreURLBone.downloadUrlFor.expiration_ would be correct but maybe it's more confusing and IMO this should be optional controllable on the bone. 

---
I am undecided.
I don't mind leaving it like that for now, and then we can think about it again later. Then we can merge the PR for the time being.

Reply:
This is correct and we leave it now as is.